pkgs/by-name/cr/crossplane-cli/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "crossplane-cli";
+  version = "1.14.3";
+
+  src = fetchFromGitHub {
+    owner = "crossplane";
+    repo = "crossplane";
+    rev = "v${version}";
+    hash = "sha256-rxN0Zi1UgQjDOkY2OJlG6826ARBzlEObJk4nDEc9784=";
+  };
+
+  vendorHash = "sha256-vkXvnEstD/czBDxmI96TIQB/L4jxhMwIS1XpHqVtxqY=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/crossplane/crossplane/internal/version.version=v${version}"
+  ];
+
+  subPackages = [ "cmd/crank" ];
+
+  postInstall = ''
+    mv $out/bin/crank $out/bin/crossplane-cli
```

Comment:
@selfuryon my suggestion is to change the binary name to `crossplane` while leaving `pname` as `crossplane-cli` just as `pname` for Helm is `kubernetes-helm` to avoid conflict with existing `pname` `helm` but binary is still just `helm`. The [Crossplane CLI Command Reference](https://docs.crossplane.io/latest/cli/command-reference/) has binary as `crossplane` in its examples.

Reply:
Oh, got that! Done